spec/system/casework/viewing_an_application/that_is_open_and_unassigned_spec.rb

```diff
@@ -345,4 +345,14 @@
       expect(page).to have_content('Home address Does not have a home address')
     end
   end
+
+  context 'with supporting evidence' do
+    it 'shows a table with supporting evidence' do
+      evidence_row = find(:xpath,
+                          "//table[@class='govuk-table app-dashboard-table govuk-!-margin-bottom-9']
+                          //tr[contains(td[1], 'test.pdf')]")
```

Comment:
Is there another way to find this table using xpath (or css)? e.g. by picking out the '3rd table in the page' or something like that?

Reply:
Probably but I'm following the pattern in the file, I think this would need refactoring for the whole file in that case